app/src/main/java/org/p2p/wallet/App.kt

```diff
@@ -45,8 +44,14 @@ import org.p2p.wallet.user.UserModule
 import org.p2p.wallet.user.repository.prices.di.TokenPricesModule
 import org.p2p.wallet.utils.SolanajTimberLogger
 import timber.log.Timber
+import kotlinx.coroutines.launch
 
 class App : Application() {
+    companion object {
+        init {
```

Comment:
maybe make this call in class P2pSdk  ? not here?
then logic will be located in one class with initialisation?